src/main/java/com/example/ajaxonboardingproject/controller/CinemaHallController.kt

```diff
@@ -23,17 +25,14 @@ data class CinemaHallController(
     @PostMapping
     fun add(
         @Valid @RequestBody requestDto: CinemaHallRequestDto
-    ): CinemaHallResponseDto {
+    ): Mono<CinemaHallResponseDto> {
         return cinemaHallService.add(cinemaHallRequestDtoMapper.mapToModel(requestDto))
             .map{ cinemaHallResponseDtoMapper.mapToDto(it) }
-            .block()!!
     }
 
     @GetMapping
-    fun getAll(): List<CinemaHallResponseDto> {
+    fun getAll(): Flux<CinemaHallResponseDto> {
```

Comment:
What the response of this endpoint?

Does Spring collect it to json array?

Reply:
everything is working fine
i think that spring works as publisher of flux and browser is obtaining messages one by one